packages/amplify_datastore/example/lib/save_model_widgets.dart

```diff
@@ -1,140 +1,155 @@
-part of sample_app;

```

Comment:
Not sure what this was about - this is the result of adding the license header and running `flutter format`.